packages/accordion/src/LionAccordion.js

```diff
@@ -43,21 +43,40 @@ export class LionAccordion extends LitElement {
           flex-direction: column;
         }
 
-        .accordion ::slotted([slot='invoker']) {
+        .accordion ::slotted([slot='accordion']) {
           margin: 0;
         }
 
-        .accordion ::slotted([slot='invoker'][expanded]) {
+        .accordion ::slotted([slot='accordion'][expanded]) {
           font-weight: bold;
         }
 
-        .accordion ::slotted([slot='content']) {
+        .accordion ::slotted([slot='accordion']) {
           margin: 0;
           visibility: hidden;
           display: none;
         }
 
-        .accordion ::slotted([slot='content'][expanded]) {
+        .accordion ::slotted([slot='accordion'][expanded]) {
+          visibility: visible;
+          display: block;
+        }
```

Comment:
All of this styling  can be gone if i'm correct.

Reply:
Correct, I removed it.